### PR TITLE
Import API documentation

### PR DESCRIPTION
**Pull request by gwarf**
Start documenting API usage. Work in progress.

### CODE REVIEW

File: source/api.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Tokens have to be generated from the `DataHub` (Onezone) interface as
+documented in :ref:`auth-token-using-web-interface` or using a command line
+call as documented hereafter.
+
+The following variables can be exported:
+
+* `OIDC_TOKEN`: OpenID Connect Access token, see
+  https://wiki.egi.eu/wiki/Federated_Cloud_OpenStack_Providers#Obtaining_an_access_token
+  for obtaining it.
```
Comment (enolfc) [inline source/api.rst]: I think this should be Check-in documentation instead of Fedcloud openstack

**Review by enolfc — DISMISSED** [review]
Looks good to me. Not sure if adding directly here how to get a token would improve the documentation (although then is one more place to keep updated in case of changes)

**Review by enolfc — COMMENTED** [review]
lesson for @enolfc: don't review things late at night, typos will be ignored.

Will do a second read later on

File: source/api.rst (hunk 1 above)
Comment (gwarf) [reply source/api.rst]: Changed the link to go to Check-in/AAI page.

**Comment by gwarf** [comment]
To be rebased and merged after #2.

**Pull request closed without merging**